DEVELOPMENT.md

```diff
@@ -11,8 +11,7 @@ This doc explains how to setup a development environment so you can get started
 1. [Create and checkout a repo fork](#checkout-your-fork)
 
 Once you meet these requirements, you can
-[start the eventing-controller](#starting-eventing-controller) and
-[install a channel provisioner](#installing-a-channel-provisioner)!
```

Comment:
it might be good to have a pointer to installing a channel (or something like that), which should explain for example how to install the `InMemoryChannel`. That can be done in a follow up PR?

Reply:
I added a section on how to install the default `InMemoryChannel` and linked to other channel implementations to have an easy starting point.